utilities/pipelines/publish/Publish-ModuleFromPathToPBR.ps1

```diff
@@ -45,69 +45,85 @@ function Publish-ModuleFromPathToPBR {
     . (Join-Path $RepoRoot 'utilities' 'pipelines' 'sharedScripts' 'Get-BRMRepositoryName.ps1')
     . (Join-Path $RepoRoot 'utilities' 'pipelines' 'sharedScripts' 'tokenReplacement' 'Convert-TokensInFileList.ps1')
 
-    $moduleFolderPath = Split-Path $TemplateFilePath -Parent
-    $moduleBicepFilePath = Join-Path $moduleFolderPath 'main.bicep'
+    $topModuleFolderPath = Split-Path $TemplateFilePath -Parent
 
-    # 1. Test if module qualifies for publishing
-    if (-not (Get-ModulesToPublish -ModuleFolderPath $moduleFolderPath)) {
-        Write-Verbose 'No changes detected. Skipping publishing' -Verbose
-        return
+    $resultSet = [ordered]@{}
+
+    # 1. Get list of all modules qualifying for publishing (updated and versioned)
+    $modulesToPublishList = Get-ModulesToPublish -ModuleFolderPath $topModuleFolderPath
```

Comment:
If I'm not mistaken, the function must be slightly changed actually. At least when looking at it locally it appears it does not check if the file exists, but just assumes it does and loads it content. This would mean the function 'should' fail for any path that is not containing a file and hence through an exception.

We'd need to add a 'Test-Path' to it before line 194 is executed
https://github.com/Azure/bicep-registry-modules/blob/a549368c4d7e57ec5541a17bbbf9a386047c01cf/utilities/pipelines/publish/helper/Get-ModulesToPublish.ps1#L194